### DISCUSSION TITLE
Topological Polar Surface Area

### DISCUSSION THREAD

**Discussion by RajFer** [Open-Systems-Pharmacology/Forum#1478] [Questions & Problems]
Hi I am new to OSP suite. Can somebody help me how to enter tPSA (topological Polar Surface Area) in PK-Sim please? 
Thanks in advance.

**Comment by Yuri05** [comment]
PK-Sim models do not rely on tPSA.
If you e.g. want to integrate a calculation of cellular permeability based on tPSA, you would need to do this in MoBi.
